src/graphql/outlier.rs

```diff
@@ -462,36 +408,16 @@ struct OutlierTotalCount {
 #[Object]
 impl OutlierTotalCount {
     /// The total number of edges.
-    async fn total_count(&self, ctx: &Context<'_>) -> Result<i64> {
-        let prefix = bincode::DefaultOptions::new().serialize(&self.model_id)?;
+    async fn total_count(&self, ctx: &Context<'_>) -> Result<usize> {
```

Comment:
I was wondering if we could use `usize` since GraphQL doesn't support a 4-byte unsigned integer type. For the total number of counts, we use `usize` in some places but `i64` elsewhere. @minshao, could you enlighten me on this?

Reply:
The return type in this instance should be refactored to either `i32` or `StringNumber` to conform to the GraphQL type system, which utilizes `Int` as a 32-bit signed integer. This adjustment is necessary to address the data precision issues outlined in issue #37. This recommendation is not specifically for this PR alone but is a broader action item we need to apply throughout the codebase to ensure consistency and prevent potential data truncation or overflow errors.